pennylane/qchem/factorization.py

```diff
@@ -124,43 +126,39 @@ def factorize(two_electron, tol_factor=1.0e-5, tol_eigval=1.0e-5):
         - Reshape the :math:`n \times n \times n \times n` two-electron tensor to a
           :math:`n^2 \times n^2` matrix where :math:`n` is the number of orbitals.
 
-        - Diagonalize the resulting matrix and keep the :math:`r` eigenvectors that have
-          corresponding eigenvalues larger than a threshold.
+        - Decompose the resulting matrix either via Cholesky decomposition or
+          via eigenvalue decomposition.
 
-        - Multiply the eigenvectors by the square root of the eigenvalues to obtain
-          matrices :math:`L^{(r)}`.
+        - In the Cholesky decomposition, we build matrices :math:`L^{(r)}` as
+          :math:`v_iv_i^{\dagger}`, where each vector :math:`v_i` is constructed
+          to explain the part of the matrix that hasn't been explained by previous
+          vectors and reduce the error term. We build :math:`r` such matrices that
+          result in an overall approximation error less than the threshold.
 
-        - Reshape the selected eigenvectors to :math:`n \times n` matrices.
+        - In the eigenvalue decomposition, we keep the :math:`r` eigenvectors that
+          have corresponding eigenvalues larger than a threshold and multiply these
+          eigenvectors by the square root of their eigenvalues to obtain
+          matrices :math:`L^{(r)}`.
```

Comment:
Why these are changed? I assume that the algorithm is the same?

Reply:
The algorithms remain the same, I am reverting these changes to keep them succint as before.